### ISSUE TITLE
Request - Have incubator eggs show how many km left out of how many it started with

### ISSUE THREAD

**Issue by Ptolemaio** [NECROBOTIO/NecroBot#4702]
hey, sorry if this is in the wrong place, but I am muted on discord for some reason.

Is there any way you can make it so that incubator eggs show for example: 2.3km left out of 10km
if it's a 10km egg?

currently it just says 2.3km left, and there's no way to tell what kind of egg it is hatching.

thanks and sorry if it's a bother. Please fix my discord permissions if you can.

Thanks

**Comment by nfettinger** [comment]
This will show on boot, but shouldnt be hard to do.

**Comment by Ptolemaio** [comment]
Thanks!

**Comment by dsmello** [comment]
good idea, and another good ting is the chose what distance egg you have preference.

**Comment by ruchern** [comment]
e8d31e0
c015817
